src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+    <React.StrictMode>
+        <App />
+    </React.StrictMode>
+);
+
+// Had an error: Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot
+
+//Had to fix it using this guide: https://stackoverflow.com/questions/71668256/deprecation-notice-reactdom-render-is-no-longer-supported-in-react-18
```

Comment:
Yes this is expected, I see the book is out of date:

<img width="744" alt="Screen Shot 2022-06-10 at 09 59 47" src="https://user-images.githubusercontent.com/58794487/173019484-49a63cb9-bc5e-411a-b2aa-894d56bcaf3f.png">

I have sent a message to the author on Twitter https://twitter.com/rwieruch